sample-iwa/src/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
Maybe put files under new `diagnostics-iwa` fodler to be consistent with other folders?